config/schedule.rb

```diff
@@ -1,3 +1,5 @@
+env 'PATH', '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
```

Comment:
Interestingly, [someone else][1] figured this out at some point, although I don't like that fix quite as much.

[1]:  https://github.com/umts/bojangles/commit/0d6edb33b0877d3cb7cfb2d4db8b87da2e9eaf3c